src/main/webapp/app/shared/util/url-utils.ts

```diff
@@ -0,0 +1,14 @@
+export const replaceUrlParams = (url: string, ...params: string[]) => {
```

Comment:
Added some tests in this [commit](https://github.com/oncokb/oncokb-transcript/pull/114/commits/c7a288ec3c60b7baf420fb6fa75b5fec20e099ea).

The function will try to replace as many params in the url.
  - If we pass in more params than the url has, then it will ignore the extras
  - If we pass in fewer params than the url, then the missing params won't be replaced in url

Reply:
I think the last scenario should resolve in exception. Practical way is to return an empty url and handle at the place where this method is called. Or create a component to redirect to main page when rerun is an empty URL's. A url without all parameters will be an invalid url anyway, it should be redirected to the main page with a warning banner?